Gems/MotionMatching/Code/Source/MotionMatchingEditorSystemComponent.h

```diff
@@ -27,6 +28,21 @@ namespace EMotionFX::MotionMatching
         MotionMatchingEditorSystemComponent();
         ~MotionMatchingEditorSystemComponent();
 
+        ////////////////////////////////////////////////////////////////////////
+        // AZTickBus interface implementation
+        int GetTickOrder() override
+        {
+            return AZ::TICK_PRE_RENDER;
+        }
+        void OnTick(float deltaTime, AZ::ScriptTimePoint time) override;
+        ////////////////////////////////////////////////////////////////////////
+
+        ////////////////////////////////////////////////////////////////////////
+        // MotionMatchingRequestBus interface implementation
+        void SetDebugDrawFeatureSchema(FeatureSchema* featureSchema) override;
+        FeatureSchema* GetDebugDrawFeatureSchema() const override;
+        ////////////////////////////////////////////////////////////////////////
```

Comment:
nit: Would suggest avoiding all the `////////////` lines and just use `AZ::TickBus overrides ...` and `MotionMatchingRequestBus overrides ...`

Reply:
Changed. Though, the `////////////` is what our python gem creation plugin actually outputs.